From 2dd875d49b1e8abe5c55e04e963868377e69895d Mon Sep 17 00:00:00 2001
From: Christopher <chrishen@hku.hk>
Date: Wed, 10 Oct 2018 18:11:26 +0800
Subject: [PATCH] install gx from source instead of pre-build package

---
 bin/Rules.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/Rules.mk b/bin/Rules.mk
index 93ee8b9..04d7bd4 100644
--- a/bin/Rules.mk
+++ b/bin/Rules.mk
@@ -20,11 +20,11 @@ endif
 
 bin/gx-v%:
 	@echo "installing gx $(@:bin/gx-%=%)"
-	bin/dist_get $(dist_root_bin) gx $@ $(@:bin/gx-%=%)
+	go get -v -u github.com/whyrusleeping/gx
 
 bin/gx-go-v%:
 	@echo "installing gx-go $(@:bin/gx-go-%=%)"
-	@bin/dist_get $(dist_root_bin) gx-go $@ $(@:bin/gx-go-%=%)
+	go get -v -u github.com/whyrusleeping/gx-go
 
 CLEAN += $(TGTS_$(d))
 include mk/footer.mk
-- 
2.16.4

